### PR TITLE
Rebased al_2019 branch: NWChem

### PR DESCRIPTION
**Pull request by vivacebelles**

### CODE REVIEW

File: qcdb/qcvars/psivardefs.py [hunk 1]
```diff
@@ -104,6 +104,10 @@ def wfn_psivars():
         args=['CUSTOM SCS-MP2 TOTAL ENERGY', 'CUSTOM SCS-MP2 CORRELATION ENERGY', 'HF TOTAL ENERGY'],
         coeff=[-1, 1, 1]))
 
+    pv0.extend(_solve_in_turn(
+        args=['CUSTOM SCS-MP2(N) TOTAL ENERGY', 'CUSTOM SCS-MP2(N) CORRELATION ENERGY', 'HF TOTAL ENERGY'],
+        coeff=[-1, 1, 1]))
+
```
Comment (loriab) [inline qcdb/qcvars/psivardefs.py]: this shouldn't be here. only SCS-MP2 gets a custom and that's to distinguish SCS-MP2 the method (with fixed coeff) from CUSTOM SCS-MP2 the user-def combo

File: tests/nwchem_tests/test_eom_ccsd_h2o.py [hunk 2]
```diff
@@ -75,17 +75,19 @@ def check_eomccsd(return_value):
 def test_1_eomccsd():
     qcdb.set_options({
         'basis'     :       '6-31g*',
+        'memory'    :       '15000 mb',
```
Comment (loriab) [inline tests/nwchem_tests/test_eom_ccsd_h2o.py]: might try reverting some of these memory lines and make sure still run. not all computers have 15gb

File: tests/nwchem_tests/test_sp-rhf-mp2-energy.py [hunk 3]
```diff
@@ -19,25 +19,29 @@ def check_mp2(return_value, is5050):
     mp2_tot = -76.230777733733
     scs_mp2_tot = -76.226922314540
     scs_corl = -0.200161577112
-    os = -0.152487590397 * 1.200000000000
-    ss = -0.051529405908 * 0.333333333333
+    os = -0.152487590397 * float(6/5)
+    ss = -0.051529405908 * float(1/3)
```
Comment (loriab) [inline tests/nwchem_tests/test_sp-rhf-mp2-energy.py]: don't need the `float(` in py3

File: qcdb/intf_nwchem/options.py [hunk 4]
```diff
@@ -268,28 +271,31 @@ def load_nwchem_defaults(options):
...
             glossary='SCF performance file true/false. Default is false.'))
 
     options.add(
         'nwchem',
         RottenOption(
-            keyword='scf_nr', default=0.1, validator=lambda x: float(x), glossary='Control Netwon-Raphson value.'))
+            keyword='scf__nr', 
+            default=0.1, 
+            validator=lambda x: float(x), 
+            glossary='Control Netwon-Raphson value.'))
```
Comment (loriab) [inline qcdb/intf_nwchem/options.py]: typo

File: tests/nwchem_tests/test_eom_ccsd_h2o.py (hunk 2 above)
Comment (vivacebelles) [reply tests/nwchem_tests/test_eom_ccsd_h2o.py]: Changed back all the memory values in NWChem tests with recent two commits (distinguish between passing/not-passing tests)

File: qcdb/qcvars/psivardefs.py [hunk 5]
```diff
@@ -118,16 +114,6 @@ def wfn_psivars():
...
 
-    # SCS(N)-MP2
-    pv0.append({
-        'form': 'SCS(N)-MP2 CORRELATION ENERGY',
-        'func': _spin_component_scaling_wsing,
-        'args': [Dm(0), Dm(1.76), 'MP2 CORRELATION ENERGY', 'MP2 SAME-SPIN CORRELATION ENERGY', 'MP2 SINGLES ENERGY']})
-    pv0.append({
-        'form': 'SCS(N)-MP2 TOTAL ENERGY',
-        'func': sum, 
-        'args': ['HF TOTAL ENERGY', 'SCS(N)-MP2 CORRELATION ENERGY']})
```
Comment (loriab) [inline qcdb/qcvars/psivardefs.py]: these shouldn't go. SCS(N)-MP2 is fine, just not the CUSTOM SCS(N)-MP2. I can just put it back.

**Pull request merged**